languages/javascript/keywords/let.md

```diff
@@ -1 +1,59 @@
 # `let`
+The `let` keyword declares and initializes a new variable. It can be optionally defined to a value.
+```javascript
+let arrow;
+arrow = "->";
+
+// or
+
+let arrow = "->";
+```
+
+Variables declared using the `let` keyword have scope in the block in which they are defined, and any contained sub-blocks. The main difference between `let` and [`var`](https://github.com/exercism/v3/blob/master/languages/javascript/keywords/var.md) is that the scope of `var` is the entire function.
```

Comment:
Link to variable, scope and use relative link for `var`.

Reply:
> is the entire function.

I don't think that's right. I think you mean when there is an enclosed function.